java/code/src/com/redhat/rhn/domain/ContentFilterEntity.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2024 SUSE LLC
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ */
+
+package com.redhat.rhn.domain;
+
+public interface ContentFilterEntity {
```

Comment:
I think it's better to add a javadoc comment here, to explain why this interface exists and that it must be implemented to use a class in a ContentFilter.

Reply:
Ah, and I forgot. I think this interface should be in package `com.redhat.rhn.domain.contentmgmt` like `ContentFilter`